Latex/inhalt/Erklärung_Praxisbeleg.tex

```diff
@@ -1,82 +1,80 @@
-
-%	Eidesstattliche Erklärung
-
-\cleardoublepage 
+%  Eidesstattliche Erklärung
+%! Dies ist eine zur Nutzung mit LaTeX angepasste Version der in Anhang 6 der Hinweise zur Anfertigung
+%! wissenschaftlicher Arbeiten an der Staatlichen Studienakademie Glauchau vorgegebenen Zustimmung
+%! zur Plagiatsprüfung.
+% Warnungen zu hbox für diese Datei deaktivieren. Die Dokumente sind so vorgegeben.
```

Comment:
ich glaub hier ist doch nochmal was durcheinander

Reply:
bezogen auf "warnungen deaktivieren" aber in der datei kommt kein `\hbadness` vor

Reply:
korrekt, das war übrig, dazu bitte #110 approven